src/rgw/librgw.cc

```diff
@@ -251,7 +251,7 @@ namespace rgw {
 
     /* XXX authorize does less here then in the REST path, e.g.,
      * the user's info is cached, but still incomplete */
-    req->log(s, "authorizing");
+    ldpp_dout(s, 2) << "authorizing" << dendl;
```

Comment:
ldpp_dout is not aesthetically pleasing, somehow, but I guess it needs to contain "dout" and it's short...